packages/node-cli/src/commands/object/import/ImportObjectsAction.js

```diff
@@ -67,21 +71,64 @@ module.exports = class ImportObjectsAction extends BaseAction {
 				}
 			}
 
-			const sdkParams = CommandUtils.extractCommandOptions(params, this._commandMetadata);
-			const executionContextForImportObjects = SdkExecutionContext.Builder.forCommand(this._commandMetadata.sdkCommand)
-				.integration()
-				.addFlags(flags)
-				.addParams(sdkParams)
-				.build();
+			let scriptIdArray = params[ANSWERS_NAMES.SCRIPT_ID].split(' ');
+			delete params[ANSWERS_NAMES.SCRIPT_ID];
+			let operationResultData = {
+				failedImports: [],
+				successfulImports: [],
+				errorImports: [],
+			};
+			let arrayOfPromises = [];
+			const numberOfSdkCalls = Math.ceil(scriptIdArray.length / NUMBER_OF_SCRIPTS);
+			const numberOfSteps = Math.ceil(numberOfSdkCalls / MAX_PARALLEL_EXECUTIONS);
 
-			const operationResult = await executeWithSpinner({
-				action: this._sdkExecutor.execute(executionContextForImportObjects),
-				message: NodeTranslationService.getMessage(MESSAGES.IMPORTING_OBJECTS),
-			});
+			for (let i = 0; i < numberOfSdkCalls; i++) {
+				const partialScriptIds = scriptIdArray.slice(i * NUMBER_OF_SCRIPTS, (i + 1) * NUMBER_OF_SCRIPTS);
+				const partialScriptIdsString = partialScriptIds.join(' ');
+				const sdkParams = CommandUtils.extractCommandOptions(params, this._commandMetadata);
+				const partialExecutionContextForImportObjects = SdkExecutionContext.Builder.forCommand(this._commandMetadata.sdkCommand)
+					.integration()
+					.addFlags(flags)
+					.addParams(sdkParams)
+					.addParam(ANSWERS_NAMES.SCRIPT_ID, partialScriptIdsString)
+					.build();
+
+				const sdkExecutor = new SdkExecutor(this._sdkPath);
+				arrayOfPromises.push(
+					sdkExecutor.execute(partialExecutionContextForImportObjects).then(function (partialOperationResult) {
+						if (partialOperationResult.status === SdkOperationResultUtils.STATUS.ERROR) {
```

Comment:
I see that each partialContext depends on the iteration, and that's a problem. But maybe we can use an arrow function so at least we don't need to use the bind(). 
Just an idea.

Reply:
I've separate it another function, but I'm still forced to use the `bind` because `partialScriptIds` depends on the execution context, not the lexical context